tools-local/tasks/FinalizeBuild.cs

```diff
@@ -92,20 +92,13 @@ public override bool Execute()
 
                 try
                 {
-                    CopyBlobs($"{Channel}/Binaries/{SharedFrameworkNugetVersion}", $"{Channel}/Binaries/Latest/");
+                    CopyBlobs($"Runtime/{SharedFrameworkNugetVersion}", $"Runtime/Latest/");
 
-                    CopyBlobs($"{Channel}/Installers/{SharedFrameworkNugetVersion}", $"{Channel}/Installers/Latest/");
-
-                    CopyBlobs($"{Channel}/Installers/{SharedHostNugetVersion}", $"{Channel}/Installers/Latest/");
-
-                    // Generate the Sharedfx Version text files
-                    List<string> versionFiles = PublishRids.Select(p => $"{p.GetMetadata("VersionFileName")}.version").ToList();
+                    CopyBlobs($"Runtime/{SharedHostNugetVersion}", $"Runtime/Latest/");
 
+                    // Generate the latest version text file
                     string sfxVersion = GetSharedFrameworkVersionFileContent();
-                    foreach(string version in versionFiles)
-                    {
-                        PublishStringToBlob(ContainerName, $"{Channel}/dnvm/latest.sharedfx.{version}", sfxVersion, "text/plain");
-                    }
+                    PublishStringToBlob(ContainerName, $"Runtime/latest.sharedfx", sfxVersion, "text/plain");
```

Comment:
This should be `Runtime/{Channel}/latest.version`.  It needs to go into a channel, and there isn't a reason to name it "sharedfx" since we are only dealing with "Runtime" stuff in this new structure.  (It was named "latest.sharedfx" before because the Sdk and the Runtime used to be publishing assets to the same place.)